.github/workflows/build-ci-buildbox-images.yaml

```diff
@@ -47,9 +49,51 @@ jobs:
       - name: Docker push the latest built image
         run: docker push $(docker images -a --format '{{.Repository}}:{{.Tag}}'| head -1)
 
+  buildbox-arm:
+    name: Build Debian ARM Buildbox
+    runs-on: ['self-hosted', 'linux', 'arm64']
```

Comment:
This is a typo, or I no longer understand our build process. I described our build process a few times in random places, so let me do the same thing again.

We build amd64 on x86 using CentOS 7 dockerfile.
We build arm64 on arm using CentOS 7 dockerfile.
We build armv7/arm on x86 using Docker-arm (so basically we cross-compile)
I don't remember how we build i386 and it's not relevant for this conversation.

My point is that we should use x86 worker to build that image.

Reply:
Seems like a typo, as `teleport.e/.github/workflows/build-linux-generic.yml` confused me. I have changed it.